src/components/pages/collectRecordFormPages/BenthicLitForm/BenthicLitObservationTable.js

```diff
@@ -156,7 +156,8 @@ const BenthicLitObservationsTable = ({
       }
 
       const handleLengthChange = (event) => {
-        const newValue = event.target.value
+        const regexNumbersOnly = new RegExp(/\D/g)
```

Comment:
a bit repetitive, could export as a const but honestly doesn't save that much code

Reply:
Maybe to be explicit `regExNumbers` (the regex itself isnt doing the restricting, the `.replace` is)

Reply:
sure, I will rename it to `regExNumbers`